pages/common/mpg321.md

```diff
@@ -0,0 +1,33 @@
+# mpg321
+
+> High Performance MPEG 1.0/2.0/2.5 Audio Player for Layer 1, 2, and 3.
```

Comment:
Perhaps this will read better?
```suggestion
> High Performance MPEG 1.0/2.0/2.5 Audio Player for layers 1, 2, and 3.
```

Reply:
No. The standard expression for MP3&Co is "Layer 1,2,3" - for example: "To play MP3 audio with XMMS, you need to add the MPEG Layer 1/2/3 Player plug-in."